app/enterprise/0.32-x/upgrades-migrations.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Upgrades and Migrations
+---
+
+# Upgrades and Migrations
+## Upgrades of Kong Enterprise Edition (EE)
+
+An upgrade (or update) is the process of changing the version of the software currently deployed to a more recent one; there is no difference with the concept of the update. We suggest following the below steps to perform a useful upgrade; notice, however, that this approach requires downtime, in case you need high availability review the Blue/Green Deployment section below.
+
+1. Download the version of Kong you are interested in upgrading to.
+2. Make a backup copy of the production database.
+3. Review the [changelog](https://docs.konghq.com/enterprise/changelog/) for the version you are upgrading to.
+4. Prepare the preproduction environment with the copy of the database taken from production.
+5. Install the newer package in preproduction.
+6. Apply any breaking changes to your preproduction environment.
+7. Run the command `kong migrations up`.
+8. Start Kong and validate the upgrade in preproduction.
+9. Schedule the maintenance window for the upgrade in production.
+10. Backup your production database in case of rollback.
+11. Install the package downloaded at step 1.
+12. Adapt your infrastructure, kong configuration file and eventual custom nginx template according to with your previous experience in preproduction.
+13. Run the `kong migrations up` command and start Kong.
+
+Once a new version is available, you are informed by a distribution list email or via your designated Customer Success Engineer (Premium subscription only). The newer version will be available in the same repository in Bintray.
```

Comment:
Please remove "(Premium subscription only)" (and same in the other version of this doc)

Reply:
Please articulate the reasons why this should be excluded